iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -354,6 +361,50 @@ public override async Task<TwinProperties> GetTwinAsync(CancellationToken cancel
             }
         }
 
+        public override void SetRefreshesOn(CancellationToken cancellationToken)
+        {
+            if (Logging.IsEnabled)
+                Logging.Enter(this, cancellationToken, nameof(SetRefreshesOn));
+
+            DateTime refreshOn = GetRefreshesOn(cancellationToken);
```

Comment:
We use both `refresh` and `refreshes`. Can we choose just one? I think `refreshes` sounds better.